src/main/java/Check.java

```diff
@@ -0,0 +1,72 @@
+public class Check {
+     //Function to check if an input has mark as the first word
+     public static boolean isMark(String input){
+        if(input.length()>=4){
+            if("mark".equalsIgnoreCase(input.substring(0,4))){
+                return true;
+            }
```

Comment:
```suggestion
     public static boolean isMark(String input) {
        if (input.length() >= 4) {
            if ("mark".equalsIgnoreCase(input.substring(0,4))) {
                return true;
            }
```
Consider appropriate adding spaces to make code more readable. I noticed the same issue appear throughout the code as well.